### ISSUE TITLE
New check: f-string containing only a single placeholder?

### ISSUE THREAD

**Issue by aureliojargas** [astral-sh/ruff#10945]
Would it be useful to have a check for an f-string whose sole content is a placeholder?

```python
a = "foo"
b = f"{a}"
print(b)
```

This could changed to `b = a` or `b = str(a)`, depending on `a` type.

```console
$ ruff --version
ruff 0.3.7

$ ruff check --isolated --select ALL foo.py
warning: `one-blank-line-before-class` (D203) and `no-blank-line-before-class` (D211) are incompatible. Ignoring `one-blank-line-before-class`.
warning: `multi-line-summary-first-line` (D212) and `multi-line-summary-second-line` (D213) are incompatible. Ignoring `multi-line-summary-second-line`.
foo.py:1:1: D100 Missing docstring in public module
foo.py:3:1: T201 `print` found
Found 2 errors.
No fixes available (1 hidden fix can be enabled with the `--unsafe-fixes` option).
$
```

**Comment by KotlinIsland** [comment]
> This could changed to `b = a` or `b = str(a)`, depending on `a` type.

or `format(a)`, it's actually kinda tricky to statically determine which.

als, should cases like:
```py
a = f"{foo:bar}"
```
be reported in favour of `format(foo, "bar")`
